fv3gfs-physics/fv3gfs/physics/stencils/microphysics.py

```diff
@@ -924,60 +938,44 @@ def sedimentation(
 
         with interval(1, -1):
 
-            if (
-                (constants.vi_fac >= 1.0e-5)
-                and (no_fall[0, 0, -1] == 0)
-                and (zt >= zt[0, 0, -1])
-            ):
-                zt = zt[0, 0, -1] - constants.dz_min
+            if (vi_fac >= 1.0e-5) and (no_fall[0, 0, -1] == 0) and (zt >= zt[0, 0, -1]):
+                zt = zt[0, 0, -1] - functions.DZ_MIN_FLIP
```

Comment:
I think this is okay because `DZ_MIN_FLIP` is just a constant used inside microphysics functions. But I was confused in the first place that this is a function. This is optional - but maybe we should change the import package as name to something other than `functions`?

Reply:
It is also true that constants are all caps, so this should also fine as it is